example/ToT/tasks/game24.py

```
@@ -0,0 +1,96 @@
+import re
+import os
+import sympy
+import pandas as pd
+from example.ToT.prompts.game24_prompts import *
```

Ruff

`from example.ToT.prompts.game24_prompts import *` used; unable to detect undefined names

(F403)